roles/ci_lvms_storage/README.md

```diff
@@ -0,0 +1,153 @@
+# ci_lvms_storage
+
+The `ci_lvms_storage` role aims to be a drop in replacement
+for `ci_local_storage`. It uses LVMS (Logical Volume Manager
+Storage) based on the TopoLVM CSI driver to dynamically
+provision local storage built from block devices on OCP
+nodes.
+
+This role requires `cifmw_lvms_disk_list` to contain a list of
+paths to block devices which already exist on OCP nodes. If
+the `cifmw_devscripts_config_overrides.vm_extradisks_list` list
+contains `"vda vdb"`, then the `devscripts` role will create these
+block devices and `cifmw_lvms_disk_list` should be set to
+`['/dev/vda', '/dev/vdb']`. This role will then pass the disk list
+to the `deviceSelector` list used by the `LVMCluster` CRD.
+
+## Privilege escalation
+
+It does not have any tasks which use become, but it does need to use
+`roles/ci_lvms_storage/templates/lvms-namespace.yaml.j2` to create
+create an OpenShift namespace where it then sets up a subscription
+as per `roles/ci_lvms_storage/templates/subscription.yaml.j2`. It
+uses the `cifmw_lvms_disk_list` list to find disks which it then wipes
+clean and adds to an LVMS cluster.
+
+## Parameters
+
+### ci-framework parameters
+
+* `cifmw_lvms`: (Boolean) Whether or not to use LVMS (default: `false`)
```

Comment:
hm, late to the party, sorry - but: to match other "boolean" like that, I'd rather go with `cifmw_use_lvms` - that would match `cifmw_use_libvirt`, `cifmw_use_devscripts` and some others we already have through the framework.

Would it make sense? It would require a mighty `sed` in this PR...

Reply:
I've made the update and we now have boolean variable `cifmw_use_lvms` in place of `cifmw_lvms`

Reply:
This actually caused a problem.

I need this var to be in `reproducer-variables.yml` on the controller node it didn't get there because of this.

https://github.com/openstack-k8s-operators/ci-framework/blob/main/roles/reproducer/tasks/configure_controller.yml#L358

Reply:
I have a plan to fix it tracked in https://issues.redhat.com/browse/OSPRH-7424